# Chemical Probes vs Molecular Probes

| Molecular                                                            | Chemical                                                       |
| -------------------------------------------------------------------- | -------------------------------------------------------------- |
| Suppress expression of protein via [[RNA Interference (RNAi)\|RNAi]] | Selective *small molecule modulators*                          |
| Irreversible                                                         | Reversible Effects                                             |
| Commonly removes/supresses whole protein                             | probes function of protein in; dose- and time-dependent manner |
| Interactome around protein + protein itself                          | controls the extent + kinetics of targeted protein modulation  |

# Chemical Probes
Purgeturbs prot function.

once a molecule is on the probes portal it is generally trusted,
dependent upon chemical matter criteria:
- no [PAINS](Pan-Assay INterference CompoundS (Pains)) element
- Available; *inactive control*
- Solubility
- Stability

and target-related criteria:Z
- Biochemical potency
- cellular potency
- selectivity
- target engagement

Probe's fundamental fitness factors (*must be adhered to*):
- in vitro potency < 100 nM
- >30-fold selectivity against other subfamilies
- on-target cellular activity < 1 uM

***Probes aren't drugs***

drug **multi-targets**
some drugs become probes if selective enough and vice versa (if multi selective)

however, little thought on drug-selectivity during development, mainly trying to manage side effects from a plethora of IP issues/risk management.

DO NOT TRUST THE CORPOS WHEN SELECTING APPROPRIATE PROBES

trust scientist-driven websites and portals:
- ==Chem Probes Portal==
- Donated Chem probes
- Gray Lab Probes
- NIH Molecular Libraries program
- Structural Genomics Consortium Chemical Probes

Score systems for [Chemical Probes Portal](http://www.chemicalprobes.org/):
uses 4 star system to recommend.
also has historic compounds:

## Historic Compounds
- over 200 compounds never to be used, ever.

## Inactive/Orthogonal Control Compounds

- Inactive is highly structurally similar, way less active than probe

- Orthogonal is structurally distinct compound but also inhibits protein of interest.
allowa to differ between **on-target** and **off-target effects**

off-target effect (lookout for):

- Inactive probe and Probe have the ==same observed phenotype==, same effect

- Orthogonal Probe and Probe have ==different phenotype/ effect not observed in Probe 2==

[[Half-Maximal Inhibitory Concentration (IC50)]]
IC50 needs to be less than 100nM,  means ==less of compound== has ==more potency.==


# [[PROTACs]]

Targeted protein degradation done with ==molecular glues== and **PRO**teolytic **TA**rgeting **C**himeras (PROTACs)

Molecular glue - binds Protein of interest and **E3 ligase**



# Molecular Probes

[[The MIQE guidelines; minimum information for publication of quantitative real-time PCR experiments.|MIQE Guidelines]] Gives a background understanding on how we manage and should use molecular probes.

Our problem with preclinical research and its irreproducability is ==***big***==:
![[Pasted image 20240305102234.png]]

In understanding molecular probes we gotta understand ==**[[The Genome]]**==

## Gene studied determined by:

- Perceived Importance
- Traceability/ease of study
- **Researcher resistance to Change** - ==scientists become attached to their research topics, refuse to branch out/create novel research==

Functions of genes analysed by most common techniques.
- [[CRISPR-Cas9]]
- [[RNA Interference (RNAi)|RNAi]] [[Gene Knockdown|(siRNA/shRNA)]]
- [[(RT-)PCR]]

> [!important] 
> We need to make sure we are using minimum run time and minimum reagent, in order to **minimize latent effect** of targeting other non-specific genes
> > 

## Error Types

- Nucleotide Sequence reagent are ==highly prone to error==!!

- Base pair sequence errors are difficult to detect (at least visually).

- Links between ==sequences== and ==descriptors== need to be verified.

Sequence errors can be typographic or mistaken identities:

| Typo          | Mistake                                 |
| ------------- | --------------------------------------- |
| additions     | Target but for *different gene*         |
| deletions     | non-targeting but with predicted target |
| substitutions | opposite of above                       |

Identity errors are *less ambiguous* and ==more concerning==
![[Pasted image 20240305105854.png]]

***[[Semi-automated fact-checking of nucleotide sequence reagents in biomedical research publications; the Seek & Blastn tool.|Seek & Blastn]]*** Definitely helps.

Manual verification of predicted errors is also important.

Most incorrect reagents from a literature review were with **RT-PCR primers**
![[Pasted image 20240305110925.png]]

Problematic papers recieved over 20k citations, ==including clinical trials?????!!!!!!== ***BRUHHHHHHHH***

TLDR (a la GPT 3.5):
  
Using wrong DNA sequences in experiments can make the research questionable. These mistakes can be reused and may lead to more wrong results. This could make other researchers try to do the same experiments, wasting their time and resources. 
It might also mean that important research gets delayed, or even worse, it could ***affect patient safety***.

Please check and verify **identities** of *all* published nucleotide sequence reagents *==before==* using them in experiments. **Do the same with cell lines**.

[[summary of probe mistakes in research]]


#chemical-probes #molecular-probes 