# Main Idea of Thesis

Another review **bruh**

> [!quote]
> "Describing and understanding
> the complete and quantitative proteome as well as its structure,
> function and dynamics is a central and fundamental challenge of biology."

## What got covered:

- Systematic Cataloging of proteins and their transformation into high-specificity assays to quantify components
- PTM analysis, focusing on **completeness of measurement** and how the community **assigns function to PTM sites**.
- MS-based techniques that can quantify thousands of proteins across large sample no's with **high reproducability**

From this it is possible to ==uncover systemic malfunctions== in protein chemistry and the biochemical ecosystem of cells and intercellular interaction which may be difficult to locate through classical research methods.

#### Doc Link: [**Mass**-**spectrometric exploration** of proteome structure and function](https://idp.nature.com/authorize/casa?redirect_uri=https://www.nature.com/articles/nature19949&casa_token=2cmgt7R3AI8AAAAA:9y54rK4FTSFBYxEokxM6zUpoRhBxX0KcLHPA-QK7tuiYboXcpFOA3HypGqxctm2F_RlOz43ug6-evY5R)


# Details/Quotes

> [!example] Proteome Exploration 
> Two strategies exist for proteome exploration:
> 1. *Conventional method* - protein isolation, followed by analysis relevant to proposed structure and function (biochemical analysis).
> 2. **Large scale, systematic proteome database generation**, can be also combined in analysis with other multi-omic data.
> 
> Both methods are greatly accelerated by developments in mass spec technique.
> 
> >[!quote]
> > "They can also **systematically identify** and **localize** ==modified amino acids== in the polypeptide chain as well as **determine** the ==composition, stoichiometry and topology== of the ==subunits== of multiprotein ==complexes== and even contribute to ***determining their structure***."

> [!statement] Data Dependent Acquisition ([[DDA]])
> - Unbiased; free from hypothesis due to not necessarily needing to identify proteins in advance.
> - All proteins can be interrogated at once
> 
> >[!quote]-
> >"Whereas the first description of a complete model proteome[6](https://www.nature.com/articles/nature19949#ref-CR6 "de Godoy, L. M. F. et al. Comprehensive mass-spectrometry-based proteome quantification of haploid versus diploid yeast. Nature 455, 1251–1254 (2008). This paper demonstrates that complete proteomes of a model organism can be obtained and quantified in different biological states.") and the identification of more than 10,000 different proteins in human cell lines[7](https://www.nature.com/articles/nature19949#ref-CR7 "Beck, M. et al. The quantitative proteome of a human cell line. Mol. Syst. Biol. 7, 549 (2011)."),[8](https://www.nature.com/articles/nature19949#ref-CR8 "Nagaraj, N. et al. Deep proteome and transcriptome mapping of a human cancer cell line. Mol. Syst. Biol. 7, 548 (2011).") were technological tours de force, a similar depth of coverage can now be achieved within hours and with minimal sample-preparation steps[9](https://www.nature.com/articles/nature19949#ref-CR9 "Hebert, A. S. et al. The one hour yeast proteome. Mol. Cell. Proteomics 13, 339–347 (2014)."),[10](https://www.nature.com/articles/nature19949#ref-CR10 "Kulak, N. A., Pichler, G., Paron, I., Nagaraj, N. & Mann, M. Minimal, encapsulated proteomic-sample processing applied to copy-number estimation in eukaryotic cells. Nature Methods 11, 319–324 (2014).")."

> [!example] PTMs and Cell Signalling
> - We are able to analyse and identify most PTMs
> - Discovery shows **more diversity, abundance** of PTMs, and their involvement/influence in cellular regulation/==cell signalling==
> - PTMs are **highly [[labile]]**
> 
> >[!quote]-
> >For these, present technology enables the identification of thousands of sites of modification and their accurate quantification between proteomic states[29](https://www.nature.com/articles/nature19949#ref-CR29 "Doll, S. & Burlingame, A. L. Mass spectrometry-based detection and assignment of protein posttranslational modifications. ACS Chem. Biol. 10, 63–71 (2015)."). The main surprise has been the number and diversity of these post-translational modifications as well as how many of them seem to be involved in cellular regulation. For example, more than 50,000 phosphorylation events on at least 75% of the proteome have been documented in a single cell line[30](https://www.nature.com/articles/nature19949#ref-CR30 "Sharma, K. et al. Ultradeep human phosphoproteome reveals a distinct regulatory nature of Tyr and Ser/Thr-based signaling. Cell Rep. 8, 1583–1594 (2014).").

![[Pasted image 20240513114835.png]]



## Related Links

## Related Reading



#### Tags: #research #reference 