# Main Idea of Thesis

**Literature review of multiple 

#### [Doc Link](https://www.sciencedirect.com/science/article/pii/S0959437X2200065X)

# Details/Quotes
> [!example] The Gist 
> - PTMs are regulate protein activity and function via chemical modification of protein residues.
> - They are a faster mechanism for phenotypic diversity than genetic variation.
> - Thus PTM divergence adds a variable to evolution, adding features for natural selection
> 
> >[!quote]-
> >"Post-translational modifications (PTMs) are chemical modifications that can regulate the activity and function of proteins. From an evolutionary perspective, they also represent a fast mechanism for the generation of phenotypic diversity and divergence."

> [!example] PTM findings for phosphorylation and acetylation
> - Experiment done investigating phosphorylation and acetylation to predict interaction effects of the modifications
> - Global protein binding is more impacted by PTM crosstalk than Local binding effects
> 
> ==Can this be generalised to other PTM conservations?==
>
> > [!quote]-
> > "The results revealed that the local effect of a PTM on binding correlates poorly with PTM conservation (in terms of sequence conservation) and that the global effect is heavily impacted by PTM crosstalk [33](https://scholar.google.com/scholar_lookup?title=Molecular%20dynamics%20shows%20complex%20interplay%20and%20long-range%20effects%20of%20post-translational%20modifications%20in%20yeast%20protein%20interactions&publication_year=2021&author=N.%20%C5%A0o%C5%A1tari%C4%87&author=V.%20van%20Noort)"

> [!example] Origin and evolution of PTM enzymes 
> Many PTM enzyme classes have an ancient origin.
> 
> >[!quote]-
> > "[Serine](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/serine "Learn more about Serine from ScienceDirect's AI-generated Topic Pages") and threonine phosphorylation, for example, is found across the three domains of life [10](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3000341), [11](https://www.nature.com/articles/s41579-019-0243-0), [12](https://www.nature.com/articles/s41597-020-0506-7)"
> 
> Consistent with this is the list of ~100 kinases predicted for the *ancestral eukaryotic kinome* from phylogenetic analysis of the kinase domain
> >[!quote]-
> >"..a global phylogenetic analysis of the kinase domain predicted a surprisingly large repertoire of ~100 kinases for the ancestral eukaryotic [kinome](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/kinome "Learn more about kinome from ScienceDirect's AI-generated Topic Pages") [23](https://www.sciencedirect.com/science/article/pii/S0959437X2200065X#bib23)"
> 
> - So many paralogues in this instance should risk spontaneous phosphorylation of unrelated ([[non-cognate]]) substrates.
> - The reason this doesn't happen is because of
> 	1. Evolution of weak specificity of D/E residues 
> 	2. Regulation through the MAPKK-dependent signalling pathway.
> 
> >[!quote]-
> >"Examination of ancestral kinase sequences revealed several charge-switching substitutions in the substrate-binding pocket, likely leading to the loss of the classic R–x–x–S/T motif and gain of weak specificity for D/E residues [24](ps://www.cell.com/cell-reports/pdf/S2211-1247(20)31591-6.pdf)"
> >
> >"...also used this approach to characterise the evolutionary transition from autoactivation to MAPKK-dependent regulation within the [MAPK](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/mitogen-activated-protein-kinase "Learn more about MAPK from ScienceDirect's AI-generated Topic Pages") family of signalling kinases [25](https://elifesciences.org/articles/38805)

> [!example] Mutation and Evolution of PTM sites 
> There is a constraint against binding affinity optimisation, caused by knock-on effects of optimisation.
> >[!quote]-
> >"Binding of the (*native*) motif (*suboptimal for cyclin binding*) ultimately leads to the ubiquitin-mediated degradation of [Sic1](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/sic1 "Learn more about Sic1 from ScienceDirect's AI-generated Topic Pages") and entry into S-phase. The suboptimal motif is conserved across orthologues, and optimising the motif by mutation significantly accelerates the kinetics of Sic1p degradation during the G-1–S transition [53](https://www.cell.com/current-biology/pdf/S0960-9822(20)31289-6.pdf)"
> 
> - **liquid-liquid-phase relationship (LLPS)** of PTM enzymes and sites is brought into focus.
> - This is because of substrate and enzyme recruitment to [[synthetic condensates]], to remove (obviate) specificity requirements.
> - LLPS has links to "phosphorylation, acetylation, [glycosylation](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/glycosylation "Learn more about glycosylation from ScienceDirect's AI-generated Topic Pages"), [citrullination](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/citrullination "Learn more about citrullination from ScienceDirect's AI-generated Topic Pages"), methylation, [sulfoxidation](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/sulfoxidation "Learn more about sulfoxidation from ScienceDirect's AI-generated Topic Pages"), and ubiquitination."
> >[!quote]-
> >



## Systematic annotation of PTM function

![[Pasted image 20240328142310.png]]
> Figure 2. Four approaches for the systematic annotation of PTM function. **(a)** **Structural bioinformatics**. Cartoon representation of PDB: 3GQI. The phosphosite at the interface is coloured in purple. **(b)** **Machine learning**. Structural (top-left), [mass spectrometry](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/mass-spectrometry "Learn more about mass spectrometry from ScienceDirect's AI-generated Topic Pages") (top-right), evolutionary (bottom-left) and specificity motif data (bottom-right) can all be used as features to train a model for the prediction of whether or not a PTM site is functional. **(c)** **Experimental fitness assay**. Introduce an [amino acid](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/amino-acids "Learn more about amino acid from ScienceDirect's AI-generated Topic Pages") mutation to nullify or mimic a PTM site and then examine fitness across several conditions. **(d)** **Mass spectrometry** (MS). ==MS workflows== can now be used to ==annotate== the ==effect== of **PTM sites** on protein **[thermal stability](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/thermostability "Learn more about thermal stability from ScienceDirect's AI-generated Topic Pages")**, **[protein turnover](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/protein-degradation "Learn more about protein turnover from ScienceDirect's AI-generated Topic Pages")**  and changes to the local protein structure surrounding the PTM site.


## Related Links

## Related Reading



#### Tags: #research #reference #proteomics #PTM 